tests/test_app_config.py

```diff
@@ -98,9 +98,9 @@ def test_app_development_config(self):
         self.assertTrue(application.config["DEBUG"])
         self.assertFalse(application.config["TESTING"])
         self.assertFalse(application.config["SQLALCHEMY_TRACK_MODIFICATIONS"])
-        self.assertEqual(
-            "postgresql:///bit_schema", application.config["SQLALCHEMY_DATABASE_URI"]
-        )
+        # self.assertEqual(
```

Comment:
Why are these commented?

Reply:
Sorry, was playing around testing different urls based on function-created or hard-coded for main db, test and remote. Just forgot to uncomment them back. will do that now 🤣